rules/best_practices.js

```diff
@@ -1,38 +1,40 @@
+'use strict';
+
 module.exports = {
-  "accessor-pairs": "error", // https://github.com/dwyl/goodparts/issues/59
-  "block-scoped-var": "error", // https://github.com/dwyl/goodparts/issues/61
-  "complexity": ["error", 10], // https://github.com/dwyl/goodparts/issues/63
-  "curly": "error", // https://github.com/dwyl/goodparts/issues/65
-  "dot-location": ["error", "property"], // https://github.com/dwyl/goodparts/issues/68
-  "eqeqeq": ["error", "always"], // https://github.com/dwyl/goodparts/issues/71
-  "no-alert": "error", // https://github.com/dwyl/goodparts/issues/72
-  "no-case-declarations": "error", // https://github.com/dwyl/goodparts/issues/74
-  "no-else-return": "error", // https://github.com/dwyl/goodparts/issues/87
-  "no-empty-pattern": "error", // https://github.com/dwyl/goodparts/issues/88
-  "no-eval": "error", // https://github.com/dwyl/goodparts/issues/89
-  "no-extra-bind": "error", // https://github.com/dwyl/goodparts/issues/90
-  "no-fallthrough": "error", // https://github.com/dwyl/goodparts/issues/95
-  "no-global-assign": "error", // https://github.com/dwyl/goodparts/issues/97
-  "no-implicit-globals": "error", // https://github.com/dwyl/goodparts/issues/110
-  "no-invalid-this": "error", // https://github.com/dwyl/goodparts/issues/115
-  "no-labels": "error", // https://github.com/dwyl/goodparts/issues/120
-  "no-loop-func": "error", // https://github.com/dwyl/goodparts/issues/125
-  "no-multi-spaces": "error", // https://github.com/dwyl/goodparts/issues/130
-  "no-new-func": "error", // https://github.com/dwyl/goodparts/issues/131
-  "no-new": "error", // https://github.com/dwyl/goodparts/issues/133
-  "no-octal": "error", // https://github.com/dwyl/goodparts/issues/137
-  "no-proto": "error", // https://github.com/dwyl/goodparts/issues/142
-  "no-return-assign": ["error", "always"], // https://github.com/dwyl/goodparts/issues/147
-  "no-self-assign": ["error", { props: true }], // https://github.com/dwyl/goodparts/issues/159
-  "no-sequences": "error", // https://github.com/dwyl/goodparts/issues/163
-  "no-throw-literal": "error", // https://github.com/dwyl/goodparts/issues/108
-  "no-unused-labels": "error", // https://github.com/dwyl/goodparts/issues/166
-  "no-useless-concat": "error", // https://github.com/dwyl/goodparts/issues/172
-  "no-unmodified-loop-condition": "error", // https://github.com/dwyl/goodparts/issues/178
-  "no-void": "error", // https://github.com/dwyl/goodparts/issues/179
-  "no-with": "error", // https://github.com/dwyl/goodparts/issues/181
-  "vars-on-top": "error", // https://github.com/dwyl/goodparts/issues/182
-  "yoda": ["error", "never", { exceptRange: true }], // https://github.com/dwyl/goodparts/issues/183
+  'accessor-pairs': 'error', // https://github.com/dwyl/goodparts/issues/59
```

Comment:
Yes 😮 . All good now though. The test fixtures are ignored by the linter for obvious reasons.